addons/mail/static/src/components/composer/composer_tests.js

```diff
@@ -1141,6 +1141,71 @@ QUnit.test('warning on send with shortcut when attempting to post message with s
     );
 });
 
+// Following test is kind of technical as it's difficult to test that fetch requests
+// are cancelled without checking attachment model fields.
+QUnit.test('[technical] remove an uploading attachment', async function (assert) {
+    assert.expect(7);
```

Comment:
Would not put `[technical]` tag. Purpose of this tag was to clearly flag tests that hardly make sense functionally but are necessary to assert good working of code. Usually test corner-cases that are unfortunate scenarios that happen due to implementation detail.

Here that's not the case. It makes total sense to be able to cancel uploading attachment: there are some available user actions to make this possible. True, visually you cannot have any proof that upload is actually being aborted. Heck this test doesn't neither: code could forge an abort controller and fake that this controller is linked to upload...

I would make one (or more) test(s) in `models/attachment_test.js` to assert presence of abort controller in temporary/uploading attachment and it's actually linked to upload RPC (don't aim for 100% proof if it's too cumbersome, of course). Then it's good to make use of this abort controller in a `components/attachment_tests.js` test.

_(just a personal taste: I prefer having 2 tests for cancel uploading, one assert UI and one assert abort signal. reason is that it's nicer to have smaller tests, and if we could define dependencies between tests, then the former is self-sustain while the 2nd require the model test)_